keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -611,14 +612,87 @@ that might indicate a serious problem?
 - A spike on failure events with keyword "failed spreadConstraint" in scheduler log.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
-No. This will be tested upon beta graduation.
 
 <!--
 Describe manual testing that was done and the outcomes.
 Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+Not yet, but it will be tested manually prior to upgrade following below steps:
```

Comment:
The scenario LGTM - please update the PRR once you run it (fine to do this after feature freeze, but please do that before graduating the feature to beta in k/k).

Reply:
Then let's merge this PR first for I may leave several days.. And I'll update this section after.

Reply:
SG